src/Symfony/phpstan.neon

```diff
@@ -0,0 +1,13 @@
+includes:
+    - vendor/phpstan/phpstan-phpunit/extension.neon
+    - vendor/phpstan/phpstan-symfony/extension.neon
+    - vendor/jangregor/phpstan-prophecy/extension.neon
```

Comment:
I wanted to avoid adding this config but without it phpstan cannot deal with prophecy mocks